tests/sites_framework/models.py

```diff
@@ -9,7 +9,6 @@ class AbstractArticle(models.Model):
     title = models.CharField(max_length=50)
```

Comment:
Just tested it and I can confirm that this is no longer needed after applying https://github.com/django/django/pull/7115.